app/templates/_has_wants_selector.html

```diff
@@ -0,0 +1,182 @@
+{% macro ppe_row(ppe) %}
```

Comment:
Helper used in a couple of places to generate a table row that represents a type of PPE